cpp/src/Slice/msbuild/slice.vcxproj.filters

```diff
@@ -17,381 +17,20 @@
   <ItemGroup>
     <ClCompile Include="..\FileTracker.cpp" />
     <ClCompile Include="..\Grammar.cpp" />
+    <ClCompile Include="..\MetadataValidator.cpp" />
     <ClCompile Include="..\Parser.cpp" />
     <ClCompile Include="..\Preprocessor.cpp" />
     <ClCompile Include="..\Scanner.cpp" />
     <ClCompile Include="..\SliceUtil.cpp" />
     <ClCompile Include="..\StringLiteralUtil.cpp" />
-    <ClCompile Include="..\FileTracker.cpp" />
```

Comment:
I don't know what was going on in this file, but it seems super outdated.
There were tons of duplicate references to files, and references to files that were removed from this folder.

If you remember back in the day, we used to have all the `PythonUtil`, `RubyUtil`, etc. files in the `Slice` folder.

Reply:
I tried opening and re-saving it from VScode but it didn't remove the duplicates.
So I thought to just remove them by hand.

Reply:
If you want, I could revert this and leave it to you.
I'm certainly not as familiar with the VS build system!